charts/internal/shoot-system-components/charts/csi-driver-node/templates/daemonset.yaml

```diff
@@ -28,6 +28,11 @@ spec:
         operator: Exists
       - effect: NoExecute
         operator: Exists
+      {{- if semverCompare ">= 1.19" .Capabilities.KubeVersion.GitVersion }}
+      securityContext:
+        seccompProfile:
+          type: RuntimeDefault
```

Comment:
Why we set to `RuntimeDefault` when the CSI drivers run in privileged mode (and will continue to run in privileged mode; there is now way to run them in non-privileged mode) and the actually used profile for privileged containers is Unconfined?

Reply:
Yes, for the `csi-driver` container this is correct. In the pod definition there are two more containers which do not run in privileged mode (`csi-liveness-probe` and `csi-node-driver-registrar`). They will have their seccomp profile changed Unconfined -> RuntimeDefault. Also specifying the `RuntimeDefault` profile on pod spec level acts as guardrails against:
1. Adding a new container and running it as `Unconfined`
2. Removing `privileged: true` from a container definition and still running as `Unconfined`

Reply:
Ok, thanks for explaining! 👍